component/widget.js

```diff
@@ -52,7 +60,28 @@ export default class Widget extends React.Component {
    * @return string[]
    */
   getReasons() {
-    return dayHelper(this.props.now)
+    return dayHelper(this.props.now, this.props.language)
+  }
+
+  getHitTextTranslation() {
+    if (this.props.language === 'pt-br')
+      return 'Aperte';
```

Comment:
Not sure if this code would be visually good when the amount of languages increases. I would suggest using a library (I'm not the best to suggest libraries for react, but maybe https://www.npmjs.com/package/react-i18next)

Reply:
@Ronkiro dei uma pesquisada nesse pacote e acho que vai dar mais trabalho configurá-lo do que implementar alguma outra solução manual. Estou com a ideia de criar um json com os textos da página traduzidos e carregá-los quando a página iniciar. O que acha?

Reply:
Acho uma boa! É suficiente, eu acho